src/main/java/tools/image/CenterPicker.java

```diff
@@ -121,21 +121,19 @@ public CenterPicker(final String mapName) {
         "File Suggestion", 1) == 0) {
       try (InputStream is = new FileInputStream(file.getPath())) {
         polygons = PointFileReaderWriter.readOneToManyPolygons(is);
-      } catch (final IOException ex1) {
-        System.out.println("Something wrong with your Polygons file: " + ex1);
-        ex1.printStackTrace();
+      } catch (final IOException e) {
+        ClientLogger.logQuietly("Something wrong with your Polygons file: " + file.getAbsolutePath(), e);
```

Comment:
I see a slight change in functionality, previously the error message was printed to stdout and the stacktrace to stderr.
Now everything is printed to stdout, IMO it should be stderr.

Reply:
True.  I did this because I wasn't sure if using `logError()` to display an error dialog here would be safe.  However, for some reason, I derped and forgot the blocking issues with the error dialog were fixed in #2883, so that shouldn't be an excuse anymore.

But, as the gist of your comment is about changing observable behavior, switching to `logError()` will do the same thing.  Do you think we need another `ClientLogger` method that effectively duplicates the legacy `logError()` behavior?  That is, simply writing the message to stderr (to force the console to be displayed) while bypassing the error dialog?  Maybe name it `logWarning()` or something?  @DanVanAtta might have an opinion on this.

For now, I'll simply replace the `logQuietly()` calls you've tagged with `logError()`.

Reply:
Actually, the question about whether displaying the console or the error dialog being more appropriate is probably moot.  Because neither `logQuietly()` nor `logError()` block, and we call `exit()` immediately afterwards, whatever gets displayed will be immediately torn down as the process exits. 😐

Reply:
I ran a quick test to see how using `logError()` would play out, and quite simply, it doesn't.  Since the map making tools run in a separate process, and they never initialize the error console/dialog, neither of these UI components are displayed as they are in the TripleA client.

That suggests we probably should not be using `ClientLogger` at all within the map making tools.  I'm going to revert the instances I added in this PR and look into replacing `ClientLogger` within the map making tools in a separate PR.